.storybook/stories/Loading/Loading.tsx

```diff
@@ -1,15 +1,18 @@
 import React, { FC } from 'react';
 import { ActivityIndicator, StyleSheet, View } from 'react-native';
 import { ColorKeys, theme } from '../../theme';
+import { Typography } from '../Typography';
 
 export type LoadingProps = {
   size?: 'small' | 'large';
   color?: ColorKeys;
+  text?: string;
 };
 
 export const Loading: FC<LoadingProps> = ({
   size = 'large',
   color = 'action-primary',
+  text = '',
```

Comment:
You don't have to add a default value, as you're rendering typography only if the text exists